infra/firebase.compose.json

```diff
@@ -4,11 +4,20 @@
     "cleanUrls": true,
     "ignore": ["firebase.json", "**/.*", "**/node_modules/**"]
   },
-  "functions": {
-    "predeploy": ["yarn build:functions"],
-    "source": "functions",
-    "runtime": "nodejs18"
-  },
+  "functions": [
+    {
+      "predeploy": ["yarn build:functions"],
+      "source": "functions",
+      "codebase": "maple",
+      "runtime": "nodejs18"
+    },
+    {
+      "predeploy": ["source /app/llm/venv/bin/activate && python3.10 -m pip install -r /app/llm/requirements.txt"],
```

Comment:
IIUC, this one should be used by docker compose and should run inside the docker enviroment (note this doesn't exactly work for reasons I don't understand)